Zend/zend_builtin_functions.stub.php

```diff
@@ -38,25 +39,28 @@ function is_a($object, string $class_name, bool $allow_string = false): bool {}
 
 function get_class_vars(string $class_name): array|false {}
 
-function get_object_vars(object $obj): array {}
+function get_object_vars(object $object): array {}
 
-function get_mangled_object_vars(object $obj): array {}
+function get_mangled_object_vars(object $object): array {}
```

Comment:
This rename for example looks good, because these functions are called get_object_vars, not get_obj_vars, and I don't think we use "obj" anywhere else (apart from internally).